recipes/replicaset.rb

```diff
@@ -21,7 +21,15 @@
 node.set['mongodb']['cluster_name'] = node['mongodb']['cluster_name']
 
 include_recipe 'mongodb::install'
-include_recipe 'mongodb::mongo_gem'
+
+ruby_block 'chef_gem_at_converge_time' do·
```

Comment:
I am almost less worried about that right now and more interested in getting this fix in.  Should that hold getting this out?